[12.0][IMP] mail_tracking: Failed messages (#447)
OCA/social: mail_tracking/readme/USAGE.rst

|cc|  **Cc**: It's a Carbon-Copy recipient. Can't know the status so is 'Unknown'

|noemail|  **No Email**: The partner doesn't have a defined email

If you want to see all tracking emails and events you can go to

* Settings > Technical > Email > Tracking emails
* Settings > Technical > Email > Tracking events

> When the message generates and 'error' status, it will apear on discuss 'Failed'

Reviewer: s/and 'error'/an 'error'

> channel. Any view that uses 'mail_thread' widget can show the failed messages

Reviewer: This is a user documentation, not technical one

```suggestion
channel. Any view with chatter can show the failed messages
```
